### DISCUSSION TITLE
Aldente 1.21.1 will not launch on M2 Pro

### DISCUSSION THREAD

**Discussion by sakete00** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#912] [Q&A] [answered]
I'm using a new MBP 14 M2 Pro and I downloaded and installed Aldente (Free) to try it out.

The app shows up in Launchpad, but when I open it, nothing happens.  I also tried installing 1.21 and nothing happens there either.

Is there a way to troubleshoot a fix for this?  Is there something obvious I'm missing?

Edit: 1.19.1 does work.

**Comment by MatthiasKerbl** [comment]
Hi @sakete00,

AlDente should work normally on your device. Did you try to follow this uninstallation guide [here](https://apphousekitchen.com/how-to-uninstall-aldente-pro/) and install AlDente again?

**Comment by Daniel14513** [comment]
Same for me, MBP 14 M1 Pro

**Comment by MatthiasKerbl** [comment]
I am currently not able to replicate this bug.
Where did you download the installation file for AlDente? On our website or on GitHub, or did you do the update within the app?

**Reply by Daniel14513** [reply]
From GitHub directly.

**Reply by MatthiasKerbl** [reply]
Did you have a previous version of AlDente installed or did you download AlDente for the first time?

**Reply by Daniel14513** [reply]
for the first time. Clean install from MacOS Ventura (latest version)

**Comment by Daniel14513** [comment, marked as answer]
okay, have now also installed 1.19.1 and after that update to the latest version. now it works for me.
